scss/_global.scss

```diff
@@ -20,35 +20,35 @@ $global-width: rem-calc(1200) !default;
 /// @type Number
 $global-lineheight: 1.5 !default;
 
-/// Colors used for buttons, callouts, links, etc. There must always be a color called `primary`.
+/// Color states used for buttons, callouts, links, etc. There must always be a color called `primary`.
 /// @type Map
-$foundation-palette: (
-  primary: #1779ba,
-  secondary: #767676,
-  success: #3adb76,
-  warning: #ffae00,
-  alert: #cc4b37,
+$foundation-states: (
```

Comment:
I would suggest that we don't have to use get-color here but to enforce that colors used here need to be declared in `$foundation-colors`this way it is possible to introduce colors "from the side" which may cause confusion, because generated elements suddenly get colors not globally defined.